tests/trustyai/drift/utils.py

```diff
@@ -453,3 +461,51 @@ def verify_upload_data_to_trustyai_service(
         client=client, token=token, trustyai_service=trustyai_service
     )
     assert expected_num_observations >= actual_num_observations
+
+
+def verify_trustyai_drift_metric_delete_request(
+    client: DynamicClient, trustyai_service: TrustyAIService, token: str, metric_name: str
+) -> None:
+    """
+    Deletes a drift metric request from the TrustyAI service and verifies that the deletion was successful.
+
+    Args:
+        client (DynamicClient): The client instance for interacting with the cluster.
+        trustyai_service (TrustyAIService): The TrustyAI service instance to interact with.
+        token (str): Authentication token for the service.
+        metric_name (str): Name of the metric to delete.
+
+    Raises:
+        ValueError: If there are no metrics to delete.
+        AssertionError: If the deletion request fails or the number of metrics after deletion is not as expected.
+    """
+    handler = TrustyAIServiceRequestHandler(token=token, service=trustyai_service, client=client)
+
+    # Get the initial number of metrics
+    drift_metrics_response = handler.get_drift_metrics(metric_name=metric_name)
+    drift_metrics_data = json.loads(drift_metrics_response.text)
+    initial_num_metrics: int = len(drift_metrics_data.get("requests", []))
+
+    # Raise an error if there are no metrics to delete
```

Comment:
this comment can be deleted as the code is self-explinatory (applies to the following comments as well)